docs/guides/connecting_to_your_data/datasource_configuration/how_to_configure_a_pandas_datasource.md

```diff
@@ -0,0 +1,355 @@
+---
+title: "How to configure a Pandas Datasource"
+---
+# [![Connect to data icon](../../../images/universal_map/Outlet-active.png)](../connect_to_data_overview.md) How to configure a Pandas Datasource
+
+import Prerequisites from '../../connecting_to_your_data/components/prerequisites.jsx'
+import UniversalMap from '/docs/images/universal_map/_universal_map.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+
+import InferredAssetDataConnector from './pandas_components/inferred_asset_data_connector.md'
+
+<UniversalMap setup='inactive' connect='active' create='inactive' validate='inactive'/>
+
+<Prerequisites>
+
+- Initialized a Data Context in your current working directory.
+- Have filesystem data available in a sub-folder of your Data Context's root directory.
+- Know [how to choose which DataConnector to use](../how_to_choose_which_dataconnector_to_use.md).
+- Know [how to choose between working with a single or multiple Batches of data](../how_to_choose_between_working_with_a_single_or_multiple_batches_of_data.md).
+
+</Prerequisites>
+
+## Steps
+
+### 1. Import necessary modules and initialize your Data Context
+
+```python
+import great_expectations as gx
+from ruamel import yaml
+
+data_context: gx.DataContext = gx.get_context()
+
+```
+The `great_expectations` module will give you access to your Data Context, which is the entry point for working with a Great Expectations project.
+
+The `yaml` module from `ruamel` will be used in validating your Datasource's configuration.  Great Expectations will use a Python dictionary representation of your Datasource configuration when you add your Datasource to your Data Context.  However, Great Expectations saves configurations as `yaml` files, so when you validate your configuration you will need to convert it from a Python dictionary to a `yaml` string, first.
+
+Your Data Context that is initialized by `get_data_context()` will be the Data Context defined in your current working directory.  It will provide you with convenience methods that we will use to validate your Datasource configuration and add your Datasource to your Great Expectations project once you have configured it.
+
+### 2. Create a new Datasource configuration.
+
+A new Datasource can be configured in Python as a dictionary with a specific set of keys.  We will build our Datasource configuration from scratch in this guide, although you can just as easily modify an existing one.
+
+To start, create an empty dictionary.  You will be populating it with keys as you go forward.
+
+At this point, the configuration for your Datasource is merely:
+
+```python
+datasource_config = {}
```

Comment:
I think we do try to have type hinting for code nowadays : 
```suggestion
datasource_config: dict = {}
```

Reply:
Thanks for the catch; updating that in the next commit.